sentences/de/light_HassLightSet.yaml

```
@@ -24,3 +24,12 @@ intents:
           - "<setzen> die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color}"
           - "die Farbe [<lichter>|<alle_lichter>] [<area>] auf {color} <setzen>"
         response: color_area
+
+        # Max/Min brightness
+      - sentences:
+          - "<setzen> [[die] Helligkeit [von|<von_dem>]] <name> auf [die|das] {brightness_level:brightness}  [ein]"
+          - "[die] Helligkeit [von|<von_dem>] <name> auf [die|das] {brightness_level:brightness} <setzen>"
```

```suggestion
          - "<setzen> [[die] Helligkeit [<von_dem>]] <name> auf [die|das] {brightness_level:brightness}  [ein]"
          - "[die] Helligkeit [<von_dem>] <name> auf [die|das] {brightness_level:brightness} <setzen>"
```

the expansion rule `<von_dem>` already covers "von" , "von dem" and "vom"